Converting a couple more functional tests to "fast"

rjrudin opened the pull request:
Also starting to convert duplicated code in ConnectedRESTQA to use ml-app-deployer

rjrudin commented on marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/ClientApiFunctionalTest.java:
```diff
@@ -1,1180 +1,1003 @@
-	 */
-	String xqyfunction(String items);
-
+package com.marklogic.client.fastfunctest;
```
Think I must have hit "format code" on this file, which is why it all appears as a diff. Sigh.

rjrudin commented on marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/functionaltest/ConnectedRESTQA.java:
```diff
@@ -1018,101 +1022,18 @@ public static void tearDownJavaRESTServer(String dbName, String[] fNames, String
-			client = null;
-		}
+	public static void setDatabaseProperties(String dbName, String prop, String propValue) {
+		ObjectNode properties = new ObjectMapper().createObjectNode();
```
ConnectedRESTQA has a lot of duplicated code on the left for making calls to the Manage API. Going to gradually convert those all over to use ManageClient from the ml-app-deployer. Will get rid of hundreds of lines of duplicated code.

BillFarber reviewed (approved):
I'm just assuming ClientApiFunctionalTest is correct.

rjrudin commented:
Yup, ClientApiFunctionalTest is still passing. I converted it to an AbstractFunctionalTest and also removed a few dozen warnings that Intellij was reporting.
